ontology/yaml/resources/LIGHTING/entity_types/LT.yaml

```diff
@@ -45,3 +45,86 @@ LT_DPC:
   implements:
   - LT
   - DPC
+
+#Entity types for CH ZRH
+LGRP_OS_LC_VADC:
```

Comment:
These seem like a merge issue. Please remove these, they are already in the master.